src/Tasks/Microsoft.Packaging.Tools/docs/trimming.md

```diff
@@ -0,0 +1,148 @@
+#Dependency Trimming
```

Comment:
It was my understanding that trimming was not a 2.0 feature.  Should this be removed?